src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/ManageMethodologyContentViewModel.cs

```diff
@@ -11,19 +13,19 @@ public class ManageMethodologyContentViewModel
     {
         public Guid Id { get; set; }
 
-        public string Slug { get; set; }
+        public string Slug { get; set; } = string.Empty;
 
-        public string Title { get; set; }
+        public string Title { get; set; } = string.Empty;
 
         [JsonConverter(typeof(StringEnumConverter))]
         public MethodologyStatus Status { get; set; }
 
         public DateTime? Published { get; set; }
 
-        public DateTime PublishScheduled { get; set; }
+        public List<ContentSectionViewModel> Content { get; set; } = new List<ContentSectionViewModel>();
```

Comment:
Can it be shorthanded to just `new()` here?

Reply:
Ach no, it's cos it's not upgraded to C#9 yet, isn't it?

Reply:
Yes we can't do that yet because it's in the Admin project which isn't updated. However you do see this shorthand in `MethodologyViewModel` of the `Content.Services` project.